packages/vue/README.md

```diff
@@ -0,0 +1,131 @@
+# @synergy-design-system/vue
+
+This package provides [vue.js](https://vuejs.org/) wrappers for [Synergy Web Components](https://github.com/SickDesignSystem/synergy/tree/main/packages/components).
+
+This package aims to provide an improved developer experience in vue applications:
+
+- Provides two way data binding and `v-model`
+- Autocompletion and Types
+- Better custom event handling of `synergy` events
+
+## Getting started
+
+### 1. Package installation
+
+Run the following steps to install the required packages.
+
+```bash
+# Install the base library and required css files
+npm install --save @synergy-design-system/vue @synergy-design-system/tokens
+```
+
+> ⚠️ Note we do **not** ship vue in this package.
+> You will have to install it by yourself first!
+
+### 2. Add the wanted theme to your application
+
+The components will not display correctly without the needed theme. Please include either light or dark theme in your application, for example in a newly installed vue application:
+
+```ts
+// src/main.ts
+// Add this line to enable the light theme for your application
+import '@synergy-design-system/tokens/dist/themes/light.css';
+
+import { createApp } from 'vue'
+import App from './App.vue'
+
+createApp(App).mount('#app')
+```
+
+### 3. Importing and rendering components
+
+You may now use the components by importing them from the `@synergy-design-system/vue` package and rendering them in your own vue components.
+
+```html
+<script lang="ts">
+// Note the name includes Vue here.
+// This is done because it would
+// clash with our native components otherwise
+import { SynVueButton } from '@synergy-design-system/vue';
+</script>
+
+<template>
+  <SynVueButton type="submit">
+    Submit me
+  </SynVueButton>
+</template>
+```
+
+### 4. Using two way databinding
+
+We support [vue two way data binding](https://vuejs.org/guide/components/v-model.html) for form components out of the box.
+You may use it in one of the following ways:
+
+```html
+<script lang="ts">
+import {
+  SynVueButton,
+  SynVueCheckbox,
+  SynVueTextArea,
+  SynVueInput,
+} from '@synergy-design-system/vue';
+
```

Comment:
I would add the "ref" import from vue here, so the whole example code could be copy pasted and it works out of the box.

```suggestion
import { ref } from 'vue';

```